tools/llm_bench/requirements.txt

```diff
@@ -1,9 +1,9 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 numpy
 --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
-openvino~=2024.5.0
-openvino-tokenizers~=2024.5.0
-openvino_genai~=2024.5.0
+openvino
+openvino-tokenizers
+openvino_genai
```

Comment:
maybe we need to `openvino_genai~=2024.5` ? it will limit to 2024.x only

Reply:
@ilya-lavrenov this is to LTS branch (2024.6.x), so limiting to 2024.6.x is expected?